### ISSUE TITLE
Minio web ui becomes really slow due to the way the transfer log is implemented

### ISSUE THREAD

**Issue by hholst80** [minio/console#1370]
**Is your feature request related to a problem? Please describe.**

I would like the upload log in the browser to have a capped list size. My Google chrome slows down to a halt and eventually the file transfer speed goes to zero because the whole browser is overrun by an extremely large file transfer history. 

**Describe the solution you'd like**

I would recommend that the file transfer list is capped at some reasonable size say 100 entries and a "... and additional X completed Y failed transfers" summary. Or even better do not have a log but provide a summary and a "in-flight transfer" stat. All completed files are simply aggregated into a stat.

**Describe alternatives you've considered**

No alternatives known to me.

**Comment by harshavardhana** [comment]
Moved to the relevant repo.

**Comment by dvaldivia** [comment]
yeah, that double slash should be fixed in the latest release

**Comment by hholst80** [comment]
In Firefox Taskmanager I saw the initial memory consumption of Minio Console ~20MB. Now it is close to 80MB. I uploaded a few tens of thousands files and before the copy was completed the I/O speed has gone down to "zero".

```
Total DISK READ :     155.68 K/s | Total DISK WRITE :      58.38 K/s
Actual DISK READ:     155.68 K/s | Actual DISK WRITE:     105.08 K/s
  TID  PRIO  USER     DISK READ  DISK WRITE  SWAPIN      IO    COMMAND
 4673 be/4 root       1888.00 K    376.00 K  0.00 %  6.65 % minio server --console-address=0.0.0.0:9001 /data
 4669 be/4 root       1616.00 K    524.00 K  0.00 %  4.16 % minio server --console-address=0.0.0.0:9001 /data
 4672 be/4 root       1024.00 K      2.13 M  0.00 %  3.21 % minio server --console-address=0.0.0.0:9001 /data
 4667 be/4 root       1760.00 K   1072.00 K  0.00 %  3.03 % minio server --console-address=0.0.0.0:9001 /data
```

**Comment by dvaldivia** [comment]
This problem got address yesterday in https://github.com/minio/console/pull/1386 which should be in the next release

**Comment by harshavardhana** [comment]
fixed by #1386
